spec/lib/profile_api_client_spec.rb

```diff
@@ -379,13 +371,13 @@ def delete_safeguarding_flag
       expect(create_school_student).to eq(response)
     end
 
-    it 'raises 422 exception if 422 status code is returned' do
-      response = { errors: [username: 'username', error: 'ERR_USER_EXISTS'] }
+    it 'raises 422 exception with the relevant message if 400 status code is returned' do
+      response = { errors: [message: 'The password is well dodgy'] }
       stub_request(:post, create_students_url)
-        .to_return(status: 422, body: response.to_json, headers: { 'content-type' => 'application/json' })
+        .to_return(status: 400, body: response.to_json, headers: { 'content-type' => 'application/json' })
 
       expect { create_school_student }.to raise_error(ProfileApiClient::Student422Error)
-        .with_message("Student not saved in Profile API (status code 422, username 'username', error 'username has already been taken')")
+        .with_message('The password is well dodgy')
```

Comment:
🤣

Reply:
Glad you liked it, thought we could do with some baked in humour!